uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/absence-tab/app.js

```diff
@@ -0,0 +1,28 @@
+define([
+  'common/angular',
+  'common/angularBootstrap',
+  'common/services/angular-date/date-format',
+  'leave-absences/shared/ui-router',
+  'leave-absences/absence-tab/modules/settings',
```

Comment:
There are also things like
```js
$logProvider.debugEnabled(settings.debug);
$resourceProvider.defaults.stripTrailingSlashes = false;
$httpProvider.defaults.headers.common["X-Requested-With"] = 'XMLHttpRequest';
```
in general, even if it's just for the log, it's better to keep things consistent across all L&A apps